clients/cli/src/copy.rs

```diff
@@ -24,75 +25,237 @@ pub fn copy(path: PathBuf, import_dest: &str, edit: bool) {
+                        }
+                    }
+                    CoreError::UiError(CreateFileAtPathError::PathContainsEmptyFile) => {
+                        if is_folder_copy {
+                            return println!(
+                                "Input destination {} contains an empty file!",
+                                import_dest
+                            );
+                        } else {
+                            exit_with(
+                                &format!(
+                                    "Input destination {} contains an empty file!",
+                                    import_dest_with_filename
+                                ),
+                                PATH_CONTAINS_EMPTY_FILE,
+                            )
+                        }
+                    }
+                    CoreError::UiError(CreateFileAtPathError::PathDoesntStartWithRoot) => {
+                        exit_with(
+                            "Import destination doesn't start with your root folder.",
+                            PATH_NO_ROOT,
+                        )
+                    }
+                    CoreError::Unexpected(msg) => exit_with(&msg, UNEXPECTED_ERROR),
+                },
+            };
 
-        match write_document(
-            &get_config(),
-            file_metadata.id,
-            content_to_import.as_bytes(),
-        ) {
-            Ok(_) => exit_with(
-                &format!("imported to {}", import_dest_with_filename),
-                SUCCESS,
-            ),
-            Err(err) => exit_with(&format!("Unexpected error: {:#?}", err), UNEXPECTED_ERROR),
+            match write_document(&get_config(), file_metadata.id, content.as_bytes()) {
+                Ok(_) => {
+                    if is_folder_copy {
+                        println!("imported to {}", import_dest_with_filename)
+                    } else {
+                        exit_with(
+                            &format!("imported to {}", import_dest_with_filename),
+                            SUCCESS,
+                        )
+                    }
+                }
+                Err(err) => exit_with(&format!("Unexpected error: {:#?}", err), UNEXPECTED_ERROR),
+            }
+        }
+        (Err(content_err), _) => {
+            if is_folder_copy {
+                println!(
+                    "Failed to read file from {}, OS error: {}",
+                    import_dest, content_err
+                )
+            } else {
+                exit_with(
+                    &format!("Failed to read file: {}", content_err),
+                    COULD_NOT_READ_OS_FILE,
+                )
+            }
+        }
+        (_, Err(path_err)) => {
+            if is_folder_copy {
+                println!(
+                    "Failed to get absolute path from {}, OS error: {}",
+                    path_err, path_err
```

Comment:
`path_err` printed for both slots where I think the first value should be `import_dest`?